libs/sdk-ui-dashboard/src/model/store/filterContext/filterContextSelectors.ts

```diff
@@ -267,6 +267,8 @@ export const selectFilterContextAttributeFilterIndexByLocalId: (
  *
  * @remarks
  * Invocations before initialization lead to invariant errors.
+ * This selector should only be used when circular filter dependencies are not allowed,
```

Comment:
Should we introduce some capability for this, check for it, and in case it's not allowed, throw invariant / return empty array and log error into the console?
What is the correct way to obtain this information, when circular dependencies are allowed?

Reply:
Yes! I totally forgot to throw it here. We have the capability for circular dependencies. I will add the condition here. 

This selector gets you all the descendants in a cascade. Maybe we could implement something in the future but we would have to take care of the cycles and also multiple parents, which makes it more complex. And we don't need it now, so I would rather not spend time on it.

Reply:
Oh I did not introduce the capability there as I managed without it. I will add it just for this case.